src/lib/Siteaccess/SiteaccessResolverInterface.php

```diff
@@ -28,7 +28,18 @@ public function getSiteaccessesForLocation(
     ): array;
 
     /**
-     * Returns complete list of siteaccesses.
+     * Returns a complete list of Site Access objects.
+     *
+     * @return \eZ\Publish\Core\MVC\Symfony\SiteAccess[]
+     */
+    public function getSiteAccessesListForLocation(
+        Location $location,
+        int $versionNo = null,
+        string $languageCode = null
```

Comment:
```suggestion
        ?string $languageCode = null
```

Reply:
@ViniTou fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1405/files/901eb9340f62841d61143946651d119acee43f81..1faec98f10e0dd188cbe13b8816146fa7acf13eb